docs/tut/4-Analysis/hfss_eig_f_convergence.csv

```diff
@@ -1,11 +1,16 @@
 "Pass []","re(Mode(1)) [g]","re(Mode(2)) [g]"
```

Comment:
this file should be completely removed by adding to gitignore

Reply:
I've taken a note to gitignore those files as a separate PR